src/app/personside/dialogpanel/sendMelding/standardTekster/StandardTekstSok.tsx

```diff
@@ -85,10 +86,15 @@ function velgTekst(
     };
 }
 
+function tilhorerSaksbehandlerKontaktsenter() {
```

Comment:
Finnes allerede en funksjon for dette `erKontaktsenter`, denne er vel lur å gjennbruke ;-)